File: module_common/src/main/java/module/common/data/entity/Comment.java
package module.common.data.entity;

/**
 * @describe: 评论实体
 * @date: 2020/3/10
 * @author: Mr Bai
 */
public class Comment {

    private String id;
    private String mediaId;

    private String userId;
    private String nickName;
    private String avatar;
    private String content;

    /*父评论的id*/
    private String commentId;
    private String updateTime;

    private String replyUserId;

    /*回复用户昵称*/
    private String replyNickName;

    /*回复数量*/
    private String countSon;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public String getCountSon() {
        return countSon;
    }

    public void setCountSon(String countSon) {
        this.countSon = countSon;
    }

    public String getMediaId() {
        return mediaId;
    }

    public void setMediaId(String mediaId) {
        this.mediaId = mediaId;
    }

    public String getUserId() {
        return userId;
    }

    public void setUserId(String userId) {
        this.userId = userId;
    }

    public String getNickName() {
        return nickName;
    }

    public void setNickName(String nickName) {
        this.nickName = nickName;
    }

    public String getAvatar() {
        return avatar;
    }

    public void setAvatar(String avatar) {
        this.avatar = avatar;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getCommentId() {
        return commentId;
    }

    public void setCommentId(String commentId) {
        this.commentId = commentId;
    }

    public String getUpdateTime() {
        return updateTime;
    }

    public void setUpdateTime(String updateTime) {
        this.updateTime = updateTime;
    }

    public String getReplyUserId() {
        return replyUserId;
    }

    public void setReplyUserId(String replyUserId) {
        this.replyUserId = replyUserId;
    }

    public String getReplyNickName() {
        return replyNickName;
    }

    public void setReplyNickName(String replyNickName) {
        this.replyNickName = replyNickName;
    }
}

File: module_common/src/main/java/module/common/data/entity/Comment.kt
package module.common.data.entity

import android.os.Parcelable
import kotlinx.parcelize.Parcelize

/**
 * @describe: 评论实体
 * @date: 2020/3/10
 * @author: Mr Bai
 */
@Parcelize
class Comment: Parcelable {
    var id: String? = null
    var mediaId: String? = null
    var userId: String? = null
    var nickName: String? = null
    var avatar: String? = null
    var content: String? = null

    /*父评论的id*/
    var commentId: String? = null
    var updateTime: String? = null
    var replyUserId: String? = null

    /*回复用户昵称*/
    var replyNickName: String? = null

    /*回复数量*/
    var countSon: String? = null
}